FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -327,13 +329,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             </msItem>                            
                             <msItem xml:id="p1_i2.14">
                                 <locus from="124ra" to="127ra"></locus>
-                                <title></title>
-                                <note>No. 14 corresponds to No. 17 in <bibl><ptr target="bm:Turaiev1905Eustathii"/><citedRange unit="page">167-177</citedRange></bibl> and in
-                                    <bibl><ptr target="bm:Turaev1906Ewostatewos"/><citedRange unit="page">92-97</citedRange></bibl>.</note>
+                                <title ref="LIT6366MEDeath"></title>
                             </msItem>
                             <msItem xml:id="p1_i2.15">
                                 <locus from="127ra" to="127va"></locus>
-                                <title></title>
+                                <title>Miracle No. 15</title>
                               <note>No. 15 being a miracle of ʾEwosṭātewos and <persName ref="PRS10048Warasin"></persName>
                                 similar to <bibl><ptr target="bm:ContiRossini1912Studi"/><citedRange unit="page">633-634</citedRange></bibl>
                                 but longer than that with an additional note on an encounter with a hermit.</note>
```

Comment:
Note that the text edited in Conti Rossini is NOT actually a miracle, but a passage from the Gadla Ewostatewos, at least in one other MS. I have opened an issue https://github.com/BetaMasaheft/Documentation/issues/2303 in order to be able to create a new ID